### PR TITLE
Allow usage of Client without any middlewares

### PR DESCRIPTION
**Pull request by eltermann**
I want to be able to make:

```python
cli = RegisteredClient(key, secret, middlewares=[])
```

### CODE REVIEW

File: epo_ops/api.py [hunk 1]
```diff
@@ -24,11 +24,9 @@ class Client(object):
     __register_path__ = 'register'
     __register_search_path__ = 'register/search'
 
-    def __init__(self, accept_type='xml', middlewares=None):
+    def __init__(self, accept_type='xml', middlewares=[Throttler()]):
```
Comment (gsong) [inline epo_ops/api.py]: Python params shouldn't be mutables, this is considered dangerous.

File: epo_ops/api.py [hunk 2]
```diff
@@ -24,11 +24,9 @@ class Client(object):
...
         self.accept_type = 'application/{0}'.format(accept_type)
         self.middlewares = middlewares
-        if not middlewares:
```
Comment (gsong) [inline epo_ops/api.py]: We can easily be more strict about the check here instead:

```
if middlewares is None:
    …
```

**Comment by gsong** [comment]
What's the motivation behind this change? Even with the anonymous user, EPO requires that throttling rules are followed. I'm trying to understand the use case of not having the throttling middleware.

**Comment by eltermann** [comment]
I'm externally controlling the throttling.

**Comment by gsong** [comment]
Got it. I'll incorporate this change, along with a few others and make a release soon. Thank you.

**Comment by gsong** [comment]
Can you please add your name to the bottom of `AUTHORS.md`?

**Comment by eltermann** [comment]
done

**Pull request merged**